docs/install/README.md

```diff
@@ -34,26 +36,342 @@ By default kind clusters run on the same Docker network which means we will have
 
 [create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) lives in the k8ssandra-operator repo. It is used to configure access to remote clusters. 
 
+## Quick Start
 
-# Single Cluster Install with Kustomize
-We will first look at a single cluster install to demonstrate that while K8ssandra Operator is designed for multi-clluster use, it can be used in a single cluster without any extran configuration.
+If you're interested in getting running as quickly as possible, there's a number of helper scripts that can be used to greatly reduce the steps to deploy a local K8ssandra cluster via kind for testing purposes.
 
-## Automated Setup
-Run `make single-up` to create a single kind cluster and deploy k8ssandra-operator along with its dependencies.  
-Check that there are two Deployments. The output should look similar to this:
+Two base `make` commands are provided that deploy a basic kind-based Kubernetes cluster(s).  These commands encapsulate the more detailed step-by-step installation instructions otherwise captured in this document.
+
+Each of these commands will do the following:
+
+* Create the kind-based cluster(s)
+* Install cert-manager to the cluster(s)
+* Install k8ssandra-operator and cass-operator to the cluster(s)
+* Install relevant CRDs to the cluster(s)
+
+Using this method, all installations are done in the `default` namespace.
```

Comment:
cert-manager is installed in its own namespace. cass-operator and k8ssandra-operator are installed in the `default` namespace.